src/btree/read.ts

```diff
@@ -107,15 +108,15 @@ export class BTreeRead {
       }
     }
 
-    yield* node.scan(this, prefix, fromKey, limit, onLimitKey);
+    yield* node.scan(this, prefix, fromKey, limit, convertEntry, onLimitKey);
```

Comment:
I think I can get rid of this one too by returning a `Promise<AsyncIterableIterator<R>>`

Reply:
I tried this and it didn't make any difference to the test. I went further and removed the yield* in tx.scan but I still got the exact same numbers